blockchain/txlookup.go

```diff
@@ -314,6 +314,6 @@ func (b *BlockChain) FetchTransactionStore(tx *btcutil.Tx) (TxStore, error) {
 	// the main chain without including fully spent trasactions in the
```

Comment:
This comment should be updated to reflect reality. The flag controls the behavior now.